azure-pipelines.yml

```diff
@@ -129,13 +129,10 @@ jobs:
         - Windows_NT_arm64
         - Windows_NT_x64
         - Windows_NT_x86
-      ${{ if in(variables['Build.DefinitionName'], 'coreclr-outerloop-gcstress0x3-gcstress0xc', 'coreclr-outerloop-gcstress-extra', 'coreclr-outerloop-r2r', 'coreclr-outerloop-r2r-extra') }}:
-        platforms:
-        - Linux_arm
-        - Linux_arm64
-        - Linux_x64
-        - Windows_NT_x64
-        - Windows_NT_x86
+      ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI') }}:
```

Comment:
Similar to above - do we actually want to support ci triggers on non-ci build definitions? I think we should be explicit about the supported build definitions. Currently any ci triggers on, say, gcstress definitions will run on all platforms.

Reply:
I don't think we ever gonna have gcstress running in CI - it's too expensive.

Reply:
If you insist I can make them to be more explicit.